libs/langgraph/langgraph/pregel/runner.py

```diff
@@ -412,9 +406,27 @@ def call(
             assert fut is not None, "writer did not return a future for call"
             # return a chained future to ensure commit() callback is called
             # before the returned future is resolved, to ensure stream order etc
-            sfut: asyncio.Future[Any] = asyncio.Future(loop=loop)
-            chain_future(fut, sfut)
-            return sfut
+            try:
+                asyncio.current_task()
```

Comment:
ooc: why this vs. `get_running_loop()`? 

And would this ever suceed but return None?